_sass/_footer.scss

```diff
@@ -29,8 +35,8 @@
   }
 }
 
-@include media-breakpoint-down('md') {
-  .footer-item {
-    display: block;
+@include media-breakpoint-up('xl') {
+  .footer {
+    margin-top: -2.5rem;
```

Comment:
That's not what I was asking. I'm asking you to modify the `<hr>` that appears above the footer on every page to not have that extra space in the first place so there isn't a need for a negative margin. Just don't modify the hr globally